src/transformers/utils/quantizers.py

```diff
@@ -0,0 +1,715 @@
+import importlib
+import inspect
+from abc import ABC, abstractmethod
+
+import torch
+from packaging import version
+
+from ..modeling_utils import is_accelerate_available
+from . import is_auto_gptq_available, is_bitsandbytes_available, is_optimum_available, logging
+
+
+if is_bitsandbytes_available():
+    import bitsandbytes as bnb
+
+    from ..pytorch_utils import Conv1D
+
+from .quantization_config import BitsAndBytesConfig, GPTQConfig, QuantizationConfigMixin, QuantizationMethod
+
+
+logger = logging.get_logger(__name__)
+
+
+class QuantizationConfigParser:
+    def __init__(self):
```

Comment:
could be nice to add some docstring here

Reply:
done